### ISSUE TITLE
[Flow] Wrong optional parameter  on Arg with default value

### ISSUE THREAD

**Issue by xanf** [dotansimha/graphql-code-generator#1097]
**Describe the bug**

**To Reproduce**
Schema
```
schema {
  query: Query
}

type Query {
  foo(field: Int = 10): Int
}
```

config
```
schema:
  - /tmp/demo.gql
generates:
  /tmp/demo.js:
    plugins:
      - flow
```

output
```
/* @flow */

export type Query = {
  foo: ?number
};

export type QueryFooArgs = {
  field?: ?number
};
```
**Expected behavior**
field on `QueryFooArgs` will not have an optional sign, since there will be always this field due to default value
```
field: number
```

Am I correct in my expectations?

**Comment by xanf** [comment]
No worry, I'll have a fork with fixes, will submit a PR soon :)

**Comment by dotansimha** [comment]
Awesome!! Thanks @xanf !!

**Comment by dotansimha** [comment]
Available in 0.16.0
